ocs_ci/ocs/resources/pod.py

```diff
@@ -375,17 +375,23 @@ def fillup_fs(self, size, fio_filename=None):
             self.io_params["filename"] = fio_filename
         self.fio_thread = self.wl_obj.run(**self.io_params)
 
-    def run_git_clone(self):
+    def run_git_clone(self, skip_install=False):
         """
         Execute git clone on a pod to simulate a Jenkins user
+
+        Args:
+            skip_install (bool): By default False, skips git package
+                installation in pod
```

Comment:
above statement is confusing with variable skip_install. I am expecting skip_install will skip the installation if the value is True.

Reply:
Done